web/src/pages/CustomerSatisfactionPage/CustomerSatisfactionPage.tsx

```diff
@@ -14,8 +14,10 @@ const CustomerSatisfactionPage = () => {
   const { currentUser } = useAuth()
   const [step, setStep] = useState(1)
   const handleNextStep = () => {
+    if (step == 4) {
```

Comment:
to be sure we can have a comparison with `>= 4`

Reply:
Yes!